source/develop/dev-process/working-with-gerrit.md

```diff
@@ -256,12 +253,18 @@ A [topic branch](http://progit.org/book/ch3-4.html) is a short-lived branch that
         url = http://gerrit.ovirt.org/p/project
         pushurl = ssh://username@gerrit.ovirt.org:29418/project.git
         fetch = +refs/heads/*:refs/remotes/gerrit/* 
+##### Installation
+
+Follow instructions [here](https://docs.openstack.org/contributors/common/setup-gerrit.html#installation)
```

Comment:
I don't know why we need the instructions. I think the only instructions needed are:

    dnf install git-review

And how to push changes, which configure git review on the first use.
See the instructions in ovirt-imageio docs:
https://github.com/oVirt/ovirt-imageio/blob/master/docs/development.md#submitting-patches